include/evmc/instructions.h

```diff
@@ -78,6 +78,7 @@ enum evmc_opcode
     OP_NUMBER = 0x43,
     OP_DIFFICULTY = 0x44,
     OP_GASLIMIT = 0x45,
+    OP_SELFBALANCE = 0x47,
```

Comment:
It is really 0x47, because chainid is 0x46: https://ethereum-magicians.org/t/opcode-repricing/3024/23?u=axic